quilt/run.go

```diff
@@ -0,0 +1,155 @@
+package main
```

Comment:
I'd also defer to others on the decision but I think this is a good observation.